docs/manual/java/releases/Migration12.md

```diff
@@ -31,7 +31,7 @@ The old `CassandraReadSideProcessor` API is still available, but is deprecated.
 
 Before migrating, we recommend that you read the documentation. You should read both [[Persistent Read-Side|ReadSide]], as well as [[Cassandra Read-Side support|ReadSideCassandra]].  Once you have done that, the following tasks need to be done to the processor class itself:
 
-* Modify your read side processor to implement [`ReadSideProcessor`](api/index.html?com/lightbend/lagom/javadsl/persistence/ReadSideProcessor.html) instead of [`CassandraReadSideProcessor`](api/index.html?com/lightbend/lagom/javadsl/persistence/cassandra/CassandraReadSideProcessor.html).
+* Modify your read side processor to implement [`ReadSideProcessor`](api/index.html?com/lightbend/lagom/javadsl/persistence/ReadSideProcessor.html) instead of `CassandraReadSideProcessor`.
 * Ensure your read side processor has both [`CassandraReadSide`](api/index.html?com/lightbend/lagom/javadsl/persistence/cassandra/CassandraReadSide.html) and [`CassandraSession`](api/index.html?com/lightbend/lagom/javadsl/persistence/cassandra/CassandraSession.html) injected into its constructor.
```

Comment:
Since `CassandraReadSideProcessor` is now deleted, this link was broken.

It's not necessary to maintain a link on the `.../java/1.6.x/...` version of the docs when: (a) the page is to migrate to `1.2` and (b) the file we are moving away from is gone.

Reply:
Related: we might even remove the `Migration12.md` page altogether since it's already available under `../java/1.3.x/...` and `../java/1.4.x/..` and `../java/1.5.x/..`.